sound/soc/sof/ipc4.c

```diff
@@ -369,6 +369,17 @@ static int sof_ipc4_tx_msg(struct snd_sof_dev *sdev, void *msg_data, size_t msg_
 	if (!msg_data)
 		return -EINVAL;
 
+	if (!no_pm) {
```

Comment:
too many negatives in "If the sent ipc msg is not for power state switch,
no_pm is set to false."

this is too confusing.

Reply:
will refine it